components/Table/Table.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC } from "react"
+
+import {
+  Table,
+  Tbody,
+  Th,
+  Thead,
+  Tr,
+  useBreakpointValue
+} from '@ironfish/ui-kit'
+import size from "byte-size"
+
+import DataRowSmall from "./DataRowSmall"
+import DataRowLarge from "./DataRowLarge"
+import RowItem from "./RowItem"
+import RowItemSpin from "./RowItemSpin"
+import { CommonTableProps } from "./types"
+import RowItemWrapperSmall from "./RowItemWrapperSmall"
+import RowItemWrapperLarge from "./RowItemWrapperLarge"
+
+size.defaultOptions({
+  precision: 2,
+})
+
+export const CommonTable: FC<CommonTableProps<any>> = ({
```

Comment:
We should avoid using `any`. I'll add this to a subsequent PR, so not a blocker here.